src/biketag/getStats.ts

```
@@ -0,0 +1,32 @@
+import { STATS_ENDPOINT } from '../common/endpoints'
+import { AvailableApis, HttpStatusCode } from '../common/enums'
+import { getSettingsPayload } from '../common/payloads'
+import { Stat } from '../common/schema'
+import { BikeTagApiResponse } from '../common/types'
+import { getApiUrl } from './helpers'
+
+export async function getStats(
+  client: BikeTagClient,
+  payload: getSettingsPayload
+): Promise<BikeTagApiResponse<Stat[]>> {
+  delete payload.source
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)